src/fonts/sans_bold_text_metrics.ts

```diff
@@ -0,0 +1,862 @@
+export const SansFontBold = {
+  glyphs: {
+    '0': {
+      x_min: 86,
+      x_max: 1038,
+      y_min: -25,
+      y_max: 1472,
+      ha: 1497,
```

Comment:
These values can be obtained. as I proposed, in run time with `canvas.measureText` which returns:
```
/** The dimensions of a piece of text in the canvas, as created by the CanvasRenderingContext2D.measureText() method. */
interface TextMetrics {
    /** Returns the measurement described below. */
    readonly actualBoundingBoxAscent: number;
    /** Returns the measurement described below. */
    readonly actualBoundingBoxDescent: number;
    /** Returns the measurement described below. */
    readonly actualBoundingBoxLeft: number;
    /** Returns the measurement described below. */
    readonly actualBoundingBoxRight: number;
    /** Returns the measurement described below. */
    readonly fontBoundingBoxAscent: number;
    /** Returns the measurement described below. */
    readonly fontBoundingBoxDescent: number;
    /** Returns the measurement described below. */
    readonly width: number;
}
```
what is the increase in size of the library witth these metrics?

Reply:
It is about 15K for the fonts we have now (the 5 numbers per glyph).  So you could think of it like a permanent/persistent cache for the canvas method.

Reply:
OK